src/plugins/WikiRouterPlugin/MediaWikiRouter.ts

```diff
@@ -0,0 +1,34 @@
+import WikiRouter from './WikiRouter';
+import { MwUtilGetUrl } from '@/@types/mediawiki';
+import {
+	inject,
+	InjectionKey,
+} from 'vue';
+
+export default class MediaWikiRouter implements WikiRouter {
+
+	private readonly getUrl: MwUtilGetUrl;
+
+	public constructor(
+		getUrl: MwUtilGetUrl,
```

Comment:
While I understand why this is injected here, it is not self-evident why we would need to inject a service that gets / generates a URL for us (and why this is better than just having a base URL as a config instead and constructing the URL ourselves). Therefore, I think it would be beneficial to anyone reading this code in the future if a comment was added to document this dependency.

Reply:
That is a good point! Though I feel I'm not at my most eloquent today. I added a first comment, but I'd be happy about ideas for how to improve it